### PR TITLE
Removes cerestation

### LINKED ISSUE #29942: Courtroom starts completely depressurized on ceres
No atmosphere and ice cold like space, on round start

### PR DESCRIPTION
**Pull request by BeeSting12**
Closes #29942 

:cl: BeeSting12
del: RIP cerestation
/:cl:

Why: Cerestation is a bad map. It is somehow the biggest map and yet it feels so empty. It takes ten minutes to get... anywhere. The majority of players dislike it despite the many fixes/changes to try to make it bearable. It's time to put the map out of its misery.

If you need any more reason, read the comments, people have brought up excellent points down there.

### CODE REVIEW

**Comment by francinum** [comment]
It's arbitrarily stretched out to "Increase high pop isolation"
also @MMMiracles

**Comment by SyruseEln** [comment]
I'm going to echo your comments in saying that Cere is just shit. Its the size of a z-level with less content that box. It makes no use of its size and ends up having the most boring, ugly, and uninspired departments out of any other station in rotation.

It is also the only station where players actively go out of their way to suicide to rid themselves of having to play it. Though I don't know if that's a meme or genuine disgust for the map. In my case it is the latter.

**Comment by BeeSting12** [comment]
@SyruseEln 
People tend to suicide on bad maps. It happened on birdboat and long hallways station (dream) a lot before both were removed.

**Comment by PraiseRatvar** [comment]
i swear there was a player poll to remove cerestation but i see its been set to invisible on statbus

@MrStonedOne any reason why? 
also would it possible to figure out how many times antags won like gangs, nukies, revs, both cults, blob on cere?

**Comment by BeeSting12** [comment]
Don't have stats for this but I'd imagine cults win fairly often because of the vast size of cere's maintenance and you can just tunnel out a cult hole. That would be okay on a smaller station but it makes it near impossible for security to do anything on a cult round on a z-level sized station when it takes 10 minutes to get anywhere.

**Comment by WJohn** [comment]
This map has been riddled with problems since its addition. It is utterly gigantic, yet only because it is nothing but hallways. It's not actually capable of supporting high pop as the rooms themselves are the same size as those on, say, box. This means that it is both incapable of high pop AND it feels empty in mid pop because people don't linger in the same hallways for long. Maps should not be 50% hallways.

**Comment by meeeeme** [comment]
I have to agree that Cere is very bland and way too big. The map needs some serious rework... Or a removal.

**Comment by lzimann** [comment]
I want to point out that the poll we had recently about Cere being removed had a positive result in favour of the removal(will update with the exact number later).

**Comment by PraiseRatvar** [comment]
@izimann what about my second question

> also would it possible to figure out how many times antags won like gangs, nukies, revs, both cults, blob on cere?

**Comment by nfreader** [comment]
>also would it possible to figure out how many times antags won like gangs, nukies, revs, both cults, blob on cere?

Theoretically, yes, but that data would need to be averaged against how many times cere has come up vs. other maps and vote weighting.

**Comment by PraiseRatvar** [comment]
another thing to point out to support the cerestation removal, admins like to change map if cere happens to come up.

**Comment by calzilla1** [comment]
Kill delta and pubby too

**Comment by Bombpants** [comment]
Go back to ministation

**Comment by BeeSting12** [comment]
Look up logs for round 74758. You'll see a good amount of complaints about the station in deadchat/OOC. I don't think anyone was defending the map and if there was someone doing so they were certainly the minority

**Comment by BeeSting12** [comment]
Frankly if we're gonna have a shit map I'd rather it be birdboat at least it was shit in the fun kinda way

**Comment by PraiseRatvar** [comment]
i expect this PR to get merged in 2 days, if so can we pay respects to cere by legitimately playing only cere for 2 days on at least one of the servers 😅

**Comment by ShizCalev** [comment]
Are you using the quantum telepads to get around? They speed things up a TON.

**Comment by PraiseRatvar** [comment]
@ShizCalev n- no..

**Comment by ShizCalev** [comment]
There are public access teleporters on every asteroid that will literally teleport you to the other end of the map if you so desired lol.

**Comment by SyruseEln** [comment]
They also have a cooldown time and cannot transport more than one person at a time. And can be camped for entrapment.

**Comment by MMMiracles** [comment]
if there was a server poll in favor of removal as @lzimann says then my only option is to get upset and yell at anyone who disagrees with me

i was getting access to a computer this week to get back into the department revamps but it seems I've taken too long. RIP

**Comment by PraiseRatvar** [comment]
I saw them when I ran around the halls but thought it was just fluff or something completely different.

@MMMiracles oh... (well the map is too big anyway)

waiting on [industristation](https://tgstation13.org/phpBB/viewtopic.php?f=11&t=11515) tbh

**Comment by ShizCalev** [comment]
@SyruseEln Camping teleporters to jump people is emergent gameplay. Shame nobody bothers to use them.

@PraiseRatvar there is a piece of paper on the table next to all of them explaining what they are/what they do/how to use them. RTFM :p

**Comment by DoctoreMad** [comment]
>(though it can be lowered ingame via RPEDs already)

On one side this is basically necessary. On the other, >Waiting for R&D to come around so you can move around the station faster

**Comment by FantasticFwoosh** [comment]
Can we keep the Titanic escape ship for the memes?

**Comment by Bombpants** [comment]
I have an idea for an improvement, either additional tube transports or some sort of ferry system that connects parts of the station together (like the monastery shuttle)

**Comment by DaxDupont** [comment]
nvidia share_2017-08-16_01-52-26

**Comment by MMMiracles** [comment]
To be fair, Birdboat is arguably the best map.

**Comment by Anonymous583** [comment]
Remove this trash map once and for all.

**Comment by MMMiracles** [comment]
if you keep that up ill make an even bigger map with twice as much empty space

**Comment by MrStonedOne** [comment]
Map too similar to box:

"It's just a reskin, it has nothing unique"

Map has something unique:

"It's shit, remove, reeeeeeee, fuck <map>"

**Comment by SyruseEln** [comment]
Yes, Cere is uniquely shit.

**Comment by MrStonedOne** [comment]
its not just cere.

Every map, that hasn't just been box but with different location layout, has been shit on for not being box enough

and every map that has just been box but with different location layout, has been shit on for being too much like box.

**Comment by DaxDupont** [comment]
Bagil seems to love meta so there's that

**Comment by SyruseEln** [comment]
I don't know. People don't kill themselves in droves on Pubby.

**Comment by lzimann** [comment]
Alright, poll results were: 
>"Remove Cerestation??"
>Yes - 290
>No - 195

**Comment by BeeSting12** [comment]
Almost a hundred people more dislike cerestation than like. I feel like that should say something about the quality of the map.

@MrStonedOne 
There is a reason that maps are modeled after box and that is because it is a map that works and has worked for a very long time as I understand it. For the most part, departments on cerestation can accommodate no more than any other station, it's just big for the sake of being big. The maintenance is heavily balanced in the favour of cults and blobs- as I said earlier a blob can win without being discovered because of how fuckhuge it is. Pubby actually is somewhat unique and I don't see the mass complaints and suicides/disconnects that I do on cerestation. There's always a few that do that sort of thing because it's not box/meta/delta and then it happens because of a strong dislike towards a map.

With all of that said, there are parts of cerestation I do like. The disposals is pretty cool, and a few other small things. However added up, the dislikes of the map vastly outweigh the likes and I believe that it warrants a removal.

**Comment by MMMiracles** [comment]
40% of voters apparently enjoy the map enough to not want it removed and I have full intentions of picking back up where I left off if this isn't getting merged, so maybe shouldn't rush to removal. 

Also watching droves of assistants killing themselves in the HOP line will never get old.

**Comment by BeeSting12** [comment]
It gets old when you realize they're not doing it for memes but because the map is legitimately shit.

**Comment by MMMiracles** [comment]
Not according to 195 people.

**Comment by BeeSting12** [comment]
290 people still want it removed- over half of the server population.

**Comment by pubby** [comment]
It's 67% in favor of removal, which is a 2/3 supermajority.

I'm accepting of Cerestation, but you have to admit the removal train isn't going to stop until it wins.

**Comment by ShizCalev** [comment]
@BeeSting12 
Please don't close my PR #29905, it has numerous fixes for anyone downstream who doesn't want to remove Cere. :(

Just remove it after it's merged (if it does end up removed.)

**Comment by BeeSting12** [comment]
@ShizCalev 
Sure

**Comment by MMMiracles** [comment]
@pubby removal train has no brakes and it's heading for the cliff

**Comment by tortellinitony** [comment]
citadel is the only downstream that might possibly want it but I'm pretty sure they only play on box station

**Comment by Tokiko1** [comment]
I am curious why you want to tear a maintained map out of the codebase instead of disabling it in the config. 

All the other removed maps either had their creator argue in favor of removal and/or were abandoned for years. This isn't the case here.

**Comment by tortellinitony** [comment]
> I am curious why you want to tear a maintained map out of the codebase instead of disabling it in the config.

Because it's shit and it takes up space

> All the other removed maps either had their creator argue in favor of removal and/or were abandoned for years. This isn't the case here.

Maybe, but the one thing they all had in common is that they were shit and it was a big meme whenever they got picked

**Comment by ShizCalev** [comment]
@tortellini a whole 1.97MB.

**Comment by MrStonedOne** [comment]
Removed things still take up space.

its why the git repo is 1.3GB

**Comment by CPTANT** [comment]
I do not dislike the concept, I dislike the implementation.

The different departments should be a lot more compact, so the departments themselves feel populated while they seem isolated from each other.

Map has several other issues too though.

File: _maps/shuttles/emergency_cere.dmm [hunk 1]
```diff
@@ -1,2122 +0,0 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (Cyberboss) [inline _maps/shuttles/emergency_cere.dmm]: Keep this

Comment (lzimann) [inline _maps/shuttles/emergency_cere.dmm]: Yea, the shuttle is fine to keep.

**Comment by nicbn** [comment]
> its why the git repo is 1.3GB
Time to fresh start boys

**Comment by Cyberboss** [comment]
Literally defeating the purpose of git, bro

File: _maps/shuttles/emergency_cere.dmm (hunk 1 above)
Comment (BeeSting12) [inline _maps/shuttles/emergency_cere.dmm]: ill go ahead and put it back, but why... it's literally the biggest shuttle ever and probably doesnt even fit in most shuttle docks

Comment (Cyberboss) [inline _maps/shuttles/emergency_cere.dmm]: That's exactly why it should stay :^)

**Comment by MMMiracles** [comment]
Just as a final note, I'd rather have this disabled server-side instead of removed entirely as I actually do want to keep working on it now that I have access to a computer again.

**Comment by Cyberboss** [comment]
@MMMiracles has a good point. This isn't an un-maintained map like mini and birdboat. The first step should be to disable it server side.

@ChangelingRain @ausops @PKPenguin321

**Comment by BeeSting12** [comment]
Just out of curiosity @MMMiracles what plans did you have for department reworks and stuff?

**Comment by BeeSting12** [comment]
I'll resolve conflicts later.

**Comment by MMMiracles** [comment]
@Beesting12 revamps were to redo the department entirely and remake it in mind with extra space being utilized/extra equipment. If you want a general idea of what I mean, Security was the first department I redid and IMO is miles better than what I originally had.

**Comment by GunHog** [comment]
@MMMiracles perhaps you could re-introduce a smaller, improved Cere? It is a shame that Cere has to die. Except for the terrible Toxins setup, it was an enjoyable map for me. Then again, the quantum pads made things quite easy for me to reach cargo, and I could remotely upgrade Medical once the miners came back. The Aux Base was in the Research asteroid, so I just had to walk across the hall to use it, making it the best map for my playstyle. 

I will miss CereStation, and I do hope that you will make a smaller but better Cere that players will at least tolerate...and avoid trying to get the shuttle called or suicide all the time.

**Comment by YakumoChen** [comment]
As security I was inclined to never leave the command asteroid because going that far from security always made it a risk. Getting a call for help from anywhere on the station outside the security asteroid was an exercise in futility because it was always too late of a prospect to go over and deal with it. Due to the population hate of Cere, whenever it rolled the server would immediately empty and the whole station would be vast and barren.

Most importantly isolation as a design is fundamentally a terrible idea given that departmental co-operation has been one of things that coders have pushed most strongly for.

**Comment by YakumoChen** [comment]
Basically the optimal playstyle of Cere is to never leave the area you start at, not only because of the vast distance between other areas but because it was an exercise in futility often to actually travel out of your area, making the whole station literally Nations.

**Comment by YakumoChen** [comment]
also the pads are never used because they don't start set up and most of the time people just stole the tools and didn't bother, and nobody wanted to be the guy to set it up.

**Comment by ShizCalev** [comment]
@YakumoChen TBH I set them up a couple rounds and literally each time I did everyone started yelling that I was a cultist because they didn't know how I was teleporting around.

**Comment by MMMiracles** [comment]
@ShizCalev that is hilarious and also really sad that the teleport pads are actually that neglected.

@YakumoChen Officers seem to have trouble with responding to calls on Ceres because they seem to forget that departmental offices exist and if anything now actually have legit purpose on a map like this. You don't even have to walk the criminal back to security after catching them, you can literally just shove them into the convenient criminal chute located in the department checkpoint and they'll be shipped off to the brig! Not too sure on the 'abandoned and empty' aspect when it rolls either, unless a lot of has changed since the couple weeks I've been away.

Anyway, I got all my shit set up and ready to go with working on the map again. Hopefully this just gets disabled server-side if the issue is bad enough so I have a bit of time to get some work pushed out.

**Comment by msgerbs** [comment]
For what it's worth @MMMiracles I agree with you. The concept is interesting and workable, but there is so much pent-up hatred that nobody even tries to learn how to actually play the map. Regardless, it definitely should not removed. If admins want to disable it, that's their call.

**Comment by Cyberboss** [comment]
Closing this, the headmins should disable the it if things are as bad as people say, but we'll continue maintaining this map for now

**Comment by BeeSting12** [comment]
I tried guys

**Pull request closed without merging**